### DISCUSSION TITLE
Support for `validateOnly` in v3

### DISCUSSION THREAD

**Discussion by undjike** [livewire/livewire#6356] [1. Help]
I'm not finding how to `validateOnly` one field in a form in v3. It's necessary to deal with wizards.

I'm aware that v3 has an API to auto-validate fields on update, but if the user doesn't type anything and validate the form, the `required` rule is not applied: an empty form just get submitted.

Can anyone help?

**Comment by undjike** [comment]
@joshhanley @PhiloNL @calebporzio is there a way to do this, please?

**Reply by PhiloNL** [reply]
The validateOnly method is still there, and works the same as v2. So I'm not sure what you mean without any code examples.

**Reply by undjike** [reply]
Oh sorry!! I was in the Form instead of the component itself.

Now, it seems like the `valideOnly` method doesn't add validation errors when the rules fail.

**Reply by sinnbeck** [reply]
I have been using it and it gave validation errors as expected. Can you show some code ?
